tests/test_qartod.py

```diff
@@ -444,6 +444,215 @@ def test_weekofyear_periods(self):
         self._run_test(cc)
 
 
+class QartodClimatologyInclusiveRangesTest(unittest.TestCase):
+    # Test that the various configuration spans (tspan, vspan, fspan, zspan) are
+    # inclusive of both endpoints.
+    def setUp(self):
+        self.cc = qartod.ClimatologyConfig()
+        self.cc.add(
+            tspan=(np.datetime64('2019-11-01'), np.datetime64('2020-02-04')),
+            fspan=(40, 70),
+            vspan=(50, 60),
+            zspan=(0, 10)
+        )
+
+    def _run_test(self, test_inputs, expected_result):
```

Comment:
Could you add a comment explaining the test_inputs and expected_result values and how they are used?

Reply:
Done.